src/test/java/seedu/address/testutil/TypicalTransaction.java

```diff
@@ -0,0 +1,76 @@
+package seedu.address.testutil;
+
+import static seedu.address.logic.commands.CommandTestUtil.VALID_GOODS_BUY_ORANGE;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_GOODS_SELL_PAPAYA;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_PRICE_BUY_ORANGE;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_PRICE_SELL_PAPAYA;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_QUANTITY_BUY_ORANGE;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_QUANTITY_SELL_PAPAYA;
+
+import seedu.address.model.transaction.Transaction;
+
+
+
+/**
+ * A utility class containing a list of {@code Poc} objects to be used in tests.
```

Comment:
For Javadocs, perhaps it would be better to add a `Represents` at the start?
So it becomes like `Represents a utility class...`

Reply:
I copied the javadocs over from `TypicalCompany.java` as well as `TypicalPoc.java`. Hence, I think it would be fine to leave it as it is for now.